lib/DDG/Spice/Chefio.pm

```diff
@@ -0,0 +1,25 @@
+package DDG::Spice::Chefio;
+# ABSTRACT: Returns package information from npm package manager's registry.
+
+use DDG::Spice;
+
+primary_example_queries "cookbook underscore";
+description "Shows an NPM package";
+name "test";
+code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/Npm.pm";
+topics "sysadmin", "programming";
+category "programming";
+attribution github  => ['https://github.com/remixz', 'zachbruggeman'],
+            twitter => ['https://twitter.com/zachbruggeman', 'zachbruggeman'];
+
+triggers startend => 'cookbook';
```

Comment:
I think we should add "chef cookbook" and "cookbook chef" to the triggers so "cpan chef cookbook" works and then `chef` would not be in the remainder.